Presentation/Renderer/IEntityFormatter.cs

```diff
@@ -0,0 +1,18 @@
+using Opcion1LosBorbotones.Domain.Entity;
+
+namespace Opcion1LosBorbotones.Presentation.Renderer;
+
+public abstract class IEntityFormatter<T> where T : IEntity
```

Comment:
Since it is an abstract class, you can give it a more meaningful name that does not follow the convention of an interface such as "ISomeName"